packages/shared/styles/components/SfBullets.scss

```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$border-radius: 50%;
```

Comment:
variables here should be prefixed by `$bullets-`, such as `$bullets-border-radius`